deploy/roles/application/tasks/main.yml

```diff
@@ -12,8 +12,6 @@
   synchronize:
     src: '{{playbook_dir}}/..'
     dest: '{{source_dir}}'
-    rsync_opts:
-      - "--filter=':- .gitignore'"
```

Comment:
According to this issue https://github.com/ansible/ansible/issues/23575#issuecomment-294022225, Ansible's `synchronize` component no longer accepts single quotes inside `rsync_opts`. Perhaps something along the lines of?
`      - '--filter=:- .gitignore'`

Reply:
Thanks @s3ththompson! This works.